modules/aws/ecs.go

```diff
@@ -18,6 +18,18 @@ func GetEcsCluster(t testing.TestingT, region string, name string) *ecs.Cluster
 
 // GetEcsClusterE fetches information about specified ECS cluster.
 func GetEcsClusterE(t testing.TestingT, region string, name string) (*ecs.Cluster, error) {
+	return GetEcsClusterWithIncludeE(t, region, name, []string{})
+}
+
+// GetEcsClusterWithInclude fetches extended information about specified ECS cluster.
+func GetEcsClusterWithInclude(t testing.TestingT, region string, name string, include []string) *ecs.Cluster {
+	clusterInfo, err := GetEcsClusterWithIncludeE(t, region, name, include)
+	require.NoError(t, err)
+	return clusterInfo
+}
+
+// GetEcsClusterWithIncludeE fetches extended information about specified ECS cluster.
```

Comment:
NIT: Could you provide a bit more detail in this comment about valid values for `include`? Or better yet, is there a set of `const` values we can use?

Reply:
Wasn't sure about the desired format of these doc blocks, and since the others were one-line only I tried to keep it somehow short.

Added a second line describing the new `include` field